src/tools/compiletest/src/header.rs

```diff
@@ -249,6 +251,7 @@ mod directives {
     pub const COMPARE_OUTPUT_LINES_BY_SUBSET: &'static str = "compare-output-lines-by-subset";
     pub const LLVM_COV_FLAGS: &'static str = "llvm-cov-flags";
     pub const FILECHECK_FLAGS: &'static str = "filecheck-flags";
+    pub const NO_AUTO_CHECK_CFG: &'static str = "no-auto-check-cfg";
```

Comment:
Could you also please document what this directive does in the dev-guide? I.e. when/why is this needed?

Reply:
The documentation will need to be very conservative, but sure I will do it together with `FALSE`.

Reply:
That's fine yeah